app/models/publications/recruitment_performance_report_scheduler.rb

```diff
@@ -1,12 +1,18 @@
 module Publications
   class RecruitmentPerformanceReportScheduler
+    def initialize(cycle_week: CycleTimetable.current_cycle_week.pred)
```

Comment:
Out of curiosity, what is `pred`?

Reply:
predecessor. It's the number before. Opposite of `succ`